python/services/github_service.py

```diff
@@ -68,9 +68,8 @@ def __init__(self, org_token: str, organisation_name: str) -> None:
         ), execute_timeout=60)
         self.organisation_name: str = organisation_name
 
-    def archive_inactive_repositories(self, repo_type_to_archive: str, last_active_cutoff_date: datetime,
-                                      allow_list: list[str]) -> None:
-        for repo in self.__get_repos_to_consider_for_archiving(repo_type_to_archive):
+    def archive_all_inactive_repositories(self, last_active_cutoff_date: datetime, allow_list: list[str]) -> None:
+        for repo in self.__get_repos_to_consider_for_archiving("all"):
```

Comment:
I thought I'd just hardwire in the value and reflect that in the name - helps simplify some of the configuration

We can always make it an injectable value in the future if we need it to be ♻️